[SC-152] Funkcjonalność dodawania gratisowych punktów

dmncp opened the pull request:
Dodałem nowy button w liście uczestników do dodawania gratisowych punktów. Uznałem, że to fajne miejsce na tę funkcjonalność dlatego, że mamy tam wygodne filtrowanie po grupach (w sumie searchbar też by się przydał). Co do samego modala to jest dość prosty, zwykły formikowy formularz z textarea i inputem typu number. W Informacji zwrotnej od razu ustawiłem jako domyślną wartość "Praca na zajęciach", żeby oszczędzić potencjalnemu prowadzącemu czas na ciągłe wpisywanie (prawdopodobnie) tej samej wyklepanej formułki.

dmncp committed:
[SC-165] generalization of json editing component and using it to edit activity

dmncp committed:
[SC-166] delete activity modal

dmncp committed:
[SC-152] modal for adding free points

kwdrt reviewed (approved):
LGTM. dobrze też że mamy teraz jedną mniej literówkę w kodzie 🥇 
Dokładnie tak o tym myślałem, good job

dmncp committed:
[SC-152] Merge remote-tracking branch 'origin/develop' into SC-152-funkcjonalnosc-dodawania-gratisowych-punktow

# Conflicts:
#	frontend/src/components/professor/ChapterDetails/ChapterDetails.js
#	frontend/src/components/professor/ChapterDetails/ChapterDetailsStyles.js
